app/graphql/resolvers/event_records_search.rb

```diff
@@ -11,15 +11,15 @@ class Resolvers::EventRecordsSearch
 
     begin
       lookahead = context[:extras][:lookahead].selection(:event_records)
+      event_records = event_records.upcoming_with_date_select if lookahead.selects?(:date)
       event_records = event_records.includes(:addresses) if lookahead.selects?(:addresses)
       event_records = event_records.includes(:categories) if lookahead.selects?(:categories)
-      event_records = event_records.includes(:dates) if lookahead.selects?(:list_date) || lookahead.selects?(:dates)
```

Comment:
braucht es die :dates nicht mehr, wenn :list_date abgerufen wird?

Reply:
good question, i forgot to mention it. `dates` are always joined with the `upcoming` scope. and with using the new `upcoming_with_date_select` scope with a sql select, the `includes` made problems.

Reply:
so if you do not want :date in graphql but want :list_date, we now don`t have include :dates, with will take much longer to load?

Reply:
hm, i do not know exactly. maybe we can add the check that `date` is not selected and re-integrate the includes. i will check that before merging.

Reply:
I have tried with:

```
if !lookahead.selects?(:date) && (lookahead.selects?(:list_date) || lookahead.selects?(:dates))
  event_records = event_records.includes(:dates)
end
```

but it takes longer to receive the data with Insomnia. so, with this result, I will leave it removed.